chess/3-web-api/web-api.md

```diff
@@ -201,13 +201,13 @@ A simple implementation of this is to have a separate Service class for each gro
 
 ```java
 public class UserService {
-	public AuthData register(UserData user) {}
-	public AuthData login(UserData user) {}
-	public void logout(AuthData auth) {}
+	public RegisterResult register(RegisterRequest registerRequest) {}
+	public LoginResult login(LoginRequest loginRequest) {}
+	public void logout(LogoutRequest logoutRequest) {}
 }
 ```
 
-Each service method receives a Request object containing all the information it needs to do its work. After performing its purpose, it returns a corresponding Result object containing the output of the method. To do their work, service classes need to make heavy use of the Model classes and Data Access classes described above.
+Each service method receives a Request object containing all the information it needs to do its work. After performing its purpose, it returns a corresponding Result object containing the output of the method. These request and result objects would contain fields pertaining to each of the endpoints above. (Note: request and result classes do not need to be specifically created to what is shown in the specifications, it is up to your design on how you want to implement these methods). To do their work, service classes need to make heavy use of the Model classes and Data Access classes described above.
```

Comment:
These additions may be better replaced with a link to the [Request and Result Classes](https://github.com/softwareconstruction240/softwareconstruction/blob/main/chess/3-web-api/web-api.md#request-and-result-classes) section which properly discusses this topic at length.

It may be appropriate to adjust the language in the final sentence of this line as well to be consistent with the direction of the PR. It makes reference to "Model classes" and "Data Access" classes, and there may be other ways to represent the relationship of all the related words.